packages/terraform-modules-public/example/variables.tf

```diff
@@ -0,0 +1,240 @@
+# Provide the next vars with a vars-file or modifying the default value
+variable google {
+  description = "The GCP Data"
+  type        = map(string)
+
+  default = {
+    project = "my-project"
+    region  = "us-west1"
+    zone    = "us-west1-a"
+  }
+}
+
+variable network_name {
+  description = "The name of the new VPC network created"
+  type        = string
+
+  default = "celo-network"
+}
+
+variable celo_env {
+  description = "The celo network to connect with"
+  type        = string
+
+  default = "baklava"
+}
+
+variable replicas {
+  description = "The replica number for each component"
+  type        = map(number)
+
+  default = {
+    validator = 1 # Also used for proxy
+    txnode    = 0
+    attestation_service = 1
+  }
+}
+
+variable instance_types {
+  description = "The instance type for each component"
+  type        = map(string)
+
+  default = {
+    validator           = "n1-standard-1"
+    proxy               = "n1-standard-1"
+    txnode              = "n1-standard-1"
+    attestation_service = "n1-standard-1"
+  }
+}
+
+variable deploy_txnode_loadbalancer {
+  description = "Deploy the Load Balancer for Transmission nodes (if txnode > 0)"
+  type        = bool
+
+  default = false
+}
+
+variable proxy_accounts {
+  description = "The account data for the proxy nodes"
+  type        = map
+
+  default = {
+    account_addresses = [
+      "0xF2...",
+    ]
+    private_keys = [
+      "1b...",
+    ]
+    account_passwords = [
+      "secret1",
+    ]
+    private_node_keys = [
```

Comment:
It looks like `private_keys` and `enodes` are the only values that are used. The README also incorrectly specifies `private_node_keys` and `enodes`

Reply:
Updated. Most of it related I had a separate folder for my development/deployment. Thanks!